src/transformers/modeling_tf_utils.py

```diff
@@ -311,9 +311,10 @@ def booleans_processing(config, **kwargs):
     final_booleans = {}
 
     if tf.executing_eagerly():
-        final_booleans["output_attentions"] = (
-            kwargs["output_attentions"] if kwargs["output_attentions"] is not None else config.output_attentions
-        )
+        final_booleans["output_attentions"] = kwargs.get("output_attentions", None)
+        if not final_booleans["output_attentions"]:
+            final_booleans["output_attentions"] = config.output_attentions
```

Comment:
This change is not related to this PR and the test makes it so that the config will override the value `output_attentions` passed if it's `False` which should not be the case. The test should be 
```suggestion
        final_booleans["output_attentions"] = kwargs.get("output_attentions", None)
        if final_booleans["output_attentions"] is None:
            final_booleans["output_attentions"] = config.output_attentions
```
and this should really be in its own PR if it's fixing a bug.

Reply:
Without this change, I don't think it'd be possible to deal with `output_attentions` argument in the TF model. 

> and this should really be in its own PR if it's fixing a bug.

What should be done for the PR then?

Reply:
Just adding the context (if not clear yet): `booleans_processing()` assumes that `output_attentions` will be in `kwargs`.
This is true so far, as we always add `output_attentions` as an argument to the model arguments.

`ConvNextModel` is the 1st (?) one that doesn't has `output_attentions` argument.

Reply:
@sgugger @ydshieh just applied the change and added a comment to explain why it's required.